<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Board</class>
 <widget class="QWidget" name="Board">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1691</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(208, 184, 124);</string>
  </property>
  <widget class="QLabel" name="hart_image">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>-20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;â™¡&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="life">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>-10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700; font-style:italic; color:#aa0000;&quot;&gt;20&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic; color:#aa0000;&quot;&gt;Cards in your hand:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; font-style:italic; color:#aa0000;&quot;&gt;Lives of wizards:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lives_of_players">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>541</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 174, 118);
font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logs">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>951</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 174, 118);
font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_4">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_5">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="card_6">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>330</y>
     <width>271</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#aa0000;&quot;&gt;Your lives:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="do_move_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select the game cards and click this button to lay them out&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Papyrus&quot;;</string>
   </property>
   <property name="text">
    <string>Do move</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
